src/widgets/widget_definitions.rs

```diff
@@ -289,6 +293,7 @@ fn build_gtk_button(bargs: &mut BuilderArgs) -> Result<gtk::Button> {
 }
 
 /// @widget image
+/// @desc A widget displaying images
```

Comment:
It only renders a single image 
```suggestion
/// @desc A widget displaying an image
```